src/main/java/seedu/address/logic/commands/client/AddClientCommand.java

```diff
@@ -83,6 +90,10 @@ public CommandResult execute(Model model, Ui ui) throws CommandException {
         ui.showClients();
         model.updateFilteredClientList(Model.PREDICATE_SHOW_ALL_CLIENTS);
 
+        if (isAddedUnderDifferentProject) {
```

Comment:
This only changes the success message; it does not solve #192 which is the bug that allows creation of a duplicate client

Reply:
Noted. Closing this PR and issuing a separate one. Thanks!